tests/system/test_analysis.py

```diff
@@ -76,7 +76,7 @@ def assert_has_job(scanomatic, job_settings):
         )
     )
     tries = 0
-    while tries < 25:
+    while tries < 50:
```

Comment:
ouch...

Reply:
it's really the first test that failed because surpassing 25, might have been done on 26th but I figured doubling is a good practice.